src/components/country-entry-card/country-entry-card.module.scss

```
@@ -0,0 +1,67 @@
+@import 'styles/ui.module';
+@import 'styles/settings';
+@import 'styles/typography-extends';
+@import 'styles/common-animations.module';
+
+.container {
+  position: absolute;
+  top: 20px;
+  left: $site-gutter;
+  padding-left: 8px;
+  height: 0px;
+  display: flex;
+  opacity: 1;
+  flex-direction: column;
+  width: $sidebar-width;
+  overflow: hidden;
+  background-color:$elephant;
+  color: white;
+  border-radius: 2px;
+  pointer-events: none;
+  @include animationFunction($propertyToAnimate: all);
+  &.open {
+    padding: 0px 8px 15px;
```

Padding seems to be a bit bigger on the design:
```suggestion
    padding: 0px 12px 20px;
```